internal/txlib/utils.go

```diff
@@ -210,3 +210,12 @@ func isValidResolutionPolicy(policy string) (IsValid bool) {
 	return false
 
 }
+
+func truncateMessage(message string) string {
+	const maxLength = 75
```

Comment:
Can we try to detect the terminal's length here? In Python this can be done with the [`termios`](https://docs.python.org/3/library/termios.html#termios.tcgetwinsize) package. There must be a way to do this in Go.

Reply:
There is this package https://pkg.go.dev/golang.org/x/crypto/ssh/terminal and it worked as expected locally, but I was getting some build errors with some other packages, so I'll check that later to be sure🙏